util/param.go

```diff
@@ -8,16 +8,20 @@ import (
 // Param is the broadcast channel data type
 type Param struct {
 	Loadpoint *int
+	Circuit   *string
```

Comment:
Analog Loadpoint dann auch hier die id.

Reply:
Brauchts das überhaupt? Zumindest aktuell brauchts die Circuitwerte noch nicht in API/UI?

Reply:
/api/state Ausgabe vom verfügbaren, genutzten und maximalen Strom pro Circuit

Reply:
Separat